__tests__/repositories/test_task_repository.py

```diff
@@ -1,8 +1,10 @@
+from datetime import datetime
```

Comment:
для создания объектов дат без времени лучше использовать 
```suggestion
from datetime import date
```